vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -32,6 +35,11 @@ open Microsoft.VisualStudio.Shell.Interop
 open Microsoft.VisualStudio.FSharp.LanguageService
 open Microsoft.VisualStudio.ComponentModelHost
 
+module LogFile =
```

Comment:
Please use ``System.Diagnostics.Trace`` - no more logging mechanisms please :)

Reply:
@dsyme It won't get merged, I promise.